src/lib/serviceRoutinesV2/getEntities.cpp

```diff
@@ -70,6 +70,11 @@ std::string getEntities
   string    idPattern  = ciP->uriParam["idPattern"];
   string    q          = ciP->uriParam["q"];
 
+  if (ciP->uriParam["options"] == "count")
```

Comment:
In postQueryContext we have the following line:

    if ((ciP->apiVersion == "v2") && (ciP->uriParam["count"] == "true"))

Now, as there are **yet another** way to ask for count, we'd need to add the following code:

     if ((ciP->apiVersion == "v2") && (ciP->uriParamOptions["count"] == true))
    {
      countP = &count;
    }

With this, it's all taken care of in postQueryContext(), so no code needed in getEntities()